### DISCUSSION TITLE
Version 8.8.0 get_netlist(recursive=True) vs Version 6.116 get_netlist_recursive()

### DISCUSSION THREAD

**Discussion by mmw-jt** [gdsfactory/gdsfactory#3185] [Q&A] [answered]
Why do I get a full list with the older version with "coordinate-like" extensions (e.g. RETICLE_5__8_3)  vs position (RETICLE_m50700000_m20025000), respectively? How can I get the old coordinate style using Version 8.8.0?

**Comment by tvt173** [comment, marked as answer]
hi @mmw-jt , you can provide a custom function to the `get_instance_name` argument of `get_netlist`, as shown below, to get instance names in the style you like

![image](https://github.com/user-attachments/assets/d32da466-149d-4121-bed6-6991bcee8605)
